lua/ror/test.lua

```diff
@@ -2,6 +2,48 @@ local config = require("ror.config").values.test
 
 local M = {}
 
+function M.non_empty_last_line(bufnr)
+  local lines = vim.fn.getbufline(bufnr, 1, '$')
+
+  local line = ""
+
+  for i = #lines, 1, -1 do
+    if lines[i] ~= "" then
+      line = lines[i]
+
+      break
+    end
+  end
+
+  return line
+end
+
+function M.verify_debugger()
+  if M.terminal_bufnr and vim.fn.ror_waiting_attach and vim.fn.bufexists(M.terminal_bufnr) then
```

Comment:
Maybe we don't need the `ror_waiting_attach` variable? Since every run it will reset it to true and I think you were meant to do a `vim.g.ror_waiting_attach` here, 
```suggestion
  if M.terminal_bufnr and vim.fn.bufexists(M.terminal_bufnr) then
```